src/custom/components/AccountDetails/index.tsx

```diff
@@ -207,7 +208,7 @@ export default function AccountDetails({
                             href={getEtherscanLink(chainId, account, 'address')}
                           >
                             <LinkIcon size={16} />
-                            <span style={{ marginLeft: '4px' }}>View on Etherscan</span>
+                            <span style={{ marginLeft: '4px' }}>{explorerLabel}</span>
```

Comment:
nit: make a styled comp and dont use inline

Reply:
Not from this PR, although I think the problem is this is a Mod, that we didn't follow the pattern. Maybe I can open an issue to do the `Mod` of this component